src/glossary.rst

```diff
@@ -714,6 +698,24 @@ Glossary
 
       * :term:`cold start`
 
+   cylc-run directory
+      The directory that contains workflows. This is, by default, ``~/cylc-run``
+      but may be configured using :cylc:conf:`global.cylc[symlink dirs]`.
+
+      See also:
+
+      * :term:`run directory`
+   
+   source directory
+      An arbitrary directory where workflows are written and stored in
```

Comment:
```suggestion
      Any directory where workflows are written and stored in
```

I wonder if it's worth considering whether a note to the effect of:
"It's good practice for run directories and source directories to be kept separate"?

Reply:
Sure, to explain the point, I am presuming you mean it is best practice to design workflows in source directory and not to design your workflows in the cylc-run directory?